core/src/main/java/com/novoda/noplayer/exoplayer/mediasource/ExoPlayerMappedTrackInfo.java

```diff
@@ -0,0 +1,25 @@
+package com.novoda.noplayer.exoplayer.mediasource;
+
+import com.google.android.exoplayer2.source.TrackGroupArray;
+import com.google.android.exoplayer2.trackselection.MappingTrackSelector;
+
+public class ExoPlayerMappedTrackInfo {
```

Comment:
Why does this class have to be `public`? I suspect it is only for testing, can we revisit?